dotnet/src/IntegrationTests/Connectors/Memory/SqliteVec/SqliteVectorStoreFixture.cs

```diff
@@ -24,6 +24,16 @@ public SqliteCollection<TKey, TRecord> GetCollection<TKey, TRecord>(
             options);
     }
 
+    public SqliteDynamicCollection GetDynamicCollection(
+        string collectionName,
+        SqliteCollectionOptions options)
+    {
+        return new SqliteDynamicCollection(
+            this.ConnectionString,
+            collectionName,
+            options);
+    }
```

Comment:
nit: expression-bodied:

```suggestion
    => new SqliteDynamicCollection(
        this.ConnectionString,
        collectionName,
        options);
```